src/test/java/com/amazon/opendistroforelasticsearch/security/SecurityAdminMigrationTests.java

```diff
@@ -21,6 +21,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import com.amazon.opendistroforelasticsearch.security.securityconf.impl.CType;
```

Comment:
Do need any of these imports? Just seeing the diff

Reply:
Good point - I'll do another scan to rid off these stray imports.